pages/artifact/review/_id.vue

```diff
@@ -157,12 +174,13 @@ export default {
         } else {
         }
       }
-    }
+    },
+    async updateTicketStatus() {
+      let response = await this.$artifactRequestStatusEndpoint.show(this.artifact.artifact.artifact_group_id)
+      this.ticket_status = response.ticket_status
+      this.artifactReleased = this.ticket_status && this.ticket_status == "released"
```

Comment:
You're missing setting artifactRequested here too right?

Reply:
[I have a longer comment about this for a future PR but this is the only thing to fix immediately for this PR]

Reply:
For this component, that is the rate and review component, we only care about whether the artifact is released or not. If it has been requested but unreleased we still do not allow the user to rate and review the artifact. Please let me know if my understanding of the spec is correct.